devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -26,8 +26,7 @@ metadata {
 
         fingerprint profileId: "0104", inClusters: "0000, 0001, 0003, 0006, 0020, 0B02, FC02", outClusters: "0019", manufacturer: "WAXMAN", model: "leakSMART Water Valve v2.10", deviceJoinName: "leakSMART Valve"
         fingerprint profileId: "0104", inClusters: "0000, 0001, 0003, 0004, 0005, 0006, 0008, 000F, 0020, 0B02", outClusters: "0003, 0019", manufacturer: "WAXMAN", model: "House Water Valve - MDL-TBD", deviceJoinName: "Waxman House Water Valve"
-        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006, 0500", outClusters: "0019", manufacturer: "", model: "E253-KR0B0ZX-HA", deviceJoinName: "Smart Gas Valve Actuator", mnmn:"SmartThings", vid: "eZEX-gas-valve"
```

Comment:
@dkirker if we no longer use custom UI metadata do we/should we remove it from the server to avoid future confusion?

Reply:
@tpmanley We can leave it on the metadata server, but probably remove it from the repository if it is there? Or, actually, maybe leave it. I am not 100% sure if there won't be any devices on the system still using it...